src/Game/GamePage.js

```diff
@@ -1,21 +1,37 @@
 import { useEffect, useMemo, useRef, useState } from "react";
 import { supabase } from "../supabase_client";
+import "./Game.css";
+import { sixSidedDice, fourSidedDice } from "../utils";
 
 const GamePage = (props) => {
   const { roomCode, playerName } = props;
   const [totalPlayerData, setTotalPlayerData] = useState([]);
   const [attackTarget, setAttackTarget] = useState("Select a target");
+  const [currentTurnPlayer, setCurrentTurnPlayer] = useState(0);
 
   const handleClick = async () => {
     const targetPlayer = totalPlayerData.find(
       (playerData) => playerData.name === attackTarget
     );
 
+    const sixDice = sixSidedDice();
+    const fourDice = fourSidedDice();
+    const diceDifference = sixDice - fourDice;
```

Comment:
```suggestion
    const diceRoll = rollSixSidedDice() - rollFourSidedDice();
```
The variable names are basically the same as the function names so theres not much point in assigning them their own variable. The function names should probably have a verb in front though.